### DISCUSSION TITLE
Urgent:Argocd UI not populating any values from repository like revision, values.yaml file .

### DISCUSSION THREAD

**Discussion by KeerthigaKrishnan** [argoproj/argo-cd#5567] [General]
Hi team, 

I have added repo and tried creating app. But it is not populating any data from repo . My repo is helm repo, it is not populating values.yaml. So I had to edit in yaml and deploy?. I doubt some permission issue, I donno where to check/ Please guide me.

**Comment by KeerthigaKrishnan** [comment]
![Screenshot 2021-02-21 at 8 12 12 AM](https://user-images.githubusercontent.com/22313326/108613985-9b662100-741c-11eb-9eaf-378102555bee.png)

Have specified repo link and revision , Not listing in Path as well not populating values.yaml. If I edit as Yaml, still values.yaml parameters not listing/ I am unable to get any logs. Please guide me, what i am missing here. I want UI to list and populate values.yaml and PATH

**Comment by KeerthigaKrishnan** [comment]
time="2021-02-21T02:46:08Z" level=info msg="manifest cache hit: &ApplicationSource{RepoURL:https://bitbucket-eng-bgl1.cisco.com/bitbucket/scm/~keerkris/argocd_test_helm.git,Path:nginx,TargetRevision:HEAD,Helm:&ApplicationSourceHelm{ValueFiles:[values.yaml values-dev.yaml],Parameters:[]HelmParameter{},ReleaseName:,Values:,FileParameters:[]HelmFileParameter{},Version:,},Kustomize:nil,Ksonnet:nil,Directory:nil,Plugin:nil,Chart:,}/03d385f31e0d1bcda14828ad751acbfc288fba
Backedn, Isee the log like this in repo-server.. Y it s not listing in UI. Please someone look into this. We are deploying in Production.

**Comment by KeerthigaKrishnan** [comment]
In checking, I found that, repository app detail call is not executing 
This is not happening

time="2021-02-21T03:06:04Z" level=info msg="received unary call /repository.RepositoryService/GetAppDetails" grpc.metho
Any reason for this. hence it is not listing anyhing in UI>

**Comment by KeerthigaKrishnan** [comment]
If, I port-forward, it is able to list path but if i configure proxy, it is not..

**Comment by wtam2018** [comment]
I don't quite get your scenario.    Did you take a look at a Helm example?  https://argoproj.github.io/argo-cd/operator-manual/cluster-bootstrapping/#helm-example
